packages/react/src/dev.tsx

```diff
@@ -67,13 +66,72 @@ function Config() {
   );
 }
 
+const getFactor = (handle?: Handle) => {
+  if (!handle || handle.type !== "email_address") {
+    throw new Error("Only use email for demo!");
+  }
+
+  const email = handle?.value;
+  const domain = email.split("@")[1];
+
+  if (domain === "slashid.dev") {
+    const oidcFactor: Factor = {
+      method: "oidc",
+      options: {
+        provider: "google",
+        client_id: import.meta.env.VITE_GOOGLE_SSO_CLIENT_ID,
+      },
+    };
+    return oidcFactor;
+  } else {
+    const emailLinkFactor: Factor = { method: "email_link" };
+    return emailLinkFactor;
+  }
+};
+
+const oidcFactors: FactorOIDC[] = [
+  {
+    method: "oidc",
+    options: {
+      provider: "google",
+      client_id: import.meta.env.VITE_GOOGLE_SSO_CLIENT_ID,
+    },
+  },
+  {
+    method: "oidc",
+    options: {
+      provider: "azuread",
+      client_id: import.meta.env.VITE_GOOGLE_SSO_CLIENT_ID,
+    },
+  },
+];
+
+const ConfiguredDynamicFlow = () => {
+  return (
+    <ConfigurationProvider text={{ "initial.oidc": "Continue with" }}>
+      <DynamicFlow
+        className="formWrapper"
+        getFactor={getFactor}
```

Comment:
Does this mean its only possible to have a single factor with `DynamicFlow`?

Is this component a drop-in replacement for `<Form>`?